src/models/Emoji.ts

```diff
@@ -0,0 +1,34 @@
+import { ObjectType, Field, Int } from "type-graphql";
+import {prop as Property, arrayProp} from "@typegoose/typegoose";
+import { Role } from "./Role";
+
+@ObjectType()
+export class Emoji {
+
+    @Field()
+    readonly id!: string;
+
+    @Property({alias: "id"})
+    readonly _id!: string;
+
+    @Field()
+    @Property({ required: true, minlength: 2, maxlength: 100})
+    name!: string;
+
+    @Field(type => [Role])
+    @arrayProp({type: Role})
+    roles!: Role[];
+
+    //jackmcadam3@gmail.com, TokakuTokisaki 
```

Comment:
This is ok this time, but in future don't actually put the code in, just state the intention to do so. Like this:
```
// Tokaku Tokisaki <jackmcadam3@gmail.com>
// TODO: Add ego property
```